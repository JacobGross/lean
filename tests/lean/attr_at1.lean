definition f (a : nat) := a + 1

attribute f [reducible] at foo

print f

section
  open foo
  print f
end

print f

namespace foo
  print f
end foo
