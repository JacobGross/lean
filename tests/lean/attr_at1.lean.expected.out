definition f : ℕ → ℕ :=
λ (a : ℕ), a + 1
definition f [reducible] : ℕ → ℕ :=
λ (a : ℕ), a + 1
definition f : ℕ → ℕ :=
λ (a : ℕ), a + 1
definition f [reducible] : ℕ → ℕ :=
λ (a : ℕ), a + 1
